### ISSUE TITLE
Repeated annotations on classes are not scanned correctly

### ISSUE THREAD

**Issue by ahie** [vaadin/flow-maven-plugin#33]
Currently the Reflections library used to scan for classes with a given annotation does not correctly identify classes that have a repeated annotation of the given type. For example, if the plugin is ran on the skeleton-starter project with an additional `HtmlImport` added to the `MainView`, the call to [`getTypesAnnotatedWith`](https://github.com/vaadin/flow-maven-plugin/blob/3f9bf299964410cbc4bb102dec60b65ecf7d8e09/src/main/java/com/vaadin/flow/plugin/common/AnnotationValuesExtractor.java#L71) no longer returns `MainView` as being annotated with `HtmlImport` at all.

This issue also causes the multiple `HtmlImport`s of `Grid` to not be picked up by the addon and consequently not added to the resulting bundle.

**Comment by SomeoneToIgnore** [comment]
Have investigated this, Reflections library is working correctly.
The thing is that repeated annotations are grouped into their container annotation by the compiler.
I.e. 
```
@HtmlImport("frontend://bower_components/vaadin-grid/vaadin-grid.html")
@HtmlImport("frontend://bower_components/vaadin-grid/vaadin-grid-column.html")
public class Test {
...
}
```
is represented as 
```
@InternalContainerAnnotationForHtml({
  @HtmlImport("frontend://bower_components/vaadin-grid/vaadin-grid.html"), 
  @HtmlImport("frontend://bower_components/vaadin-grid/vaadin-grid-column.html")
}
public class Test {
...
}
```
in the bytecode, where `InternalContainerAnnotationForHtml` is the annotation class, specified as `@Repeatable` value in the `HtmlImport` declaration.

So, instead of just searching for `HtmlImport`, we should also search for `InternalContainerAnnotationForHtml` annotation.

But recent commit (https://github.com/vaadin/flow/commit/61998bf7af70848d00c83017b7a7f8b24c5bf447) have changed the name of the `InternalContainerAnnotationForHtml`: it is `HtmlImport.Container` now.

But we cannot just search for the new annotation now because all current component jars are compiled with the old `flow-server`, with the old annotation because new one is not released yet.

So, to fix the issue, we need to wait until 
- [ ] new `flow-server` is released, containing the commit mentioned above
- [ ] all existing components are recompiled against with this new version of `flow-server` and released

If we update the plugin code before that, it will fail with `ClassNotFoundException` when tries to search for new annotation (on the projects that use old `flow-server` version) or won't find any data (on the projects that use new `flow-server` and old components).

One extra thing to keep in mind is that there is a plan to change package names again in future.
If the annotations get affected by this, it will cause the same issues again (exceptions or missing data)

**Comment by ahie** [comment]
Closed by #34.
